pipe-cli/src/utilities/datastorage_operations.py

```diff
@@ -802,11 +818,22 @@ def _transfer_item(cls, item, manager, source_wrapper, destination_wrapper, tran
         full_path = item[1]
         relative_path = item[2]
         size = item[3]
+
+        # Unused here but added for visibility of the field
+        source_modification_date = None
+        if len(item) >= 5:
+            source_modification_date = item[4]
+
+        if len(item) >= 6:
```

Comment:
it looks like we can extract method here and reuse it at [303](https://github.com/epam/cloud-pipeline/pull/3894/files#diff-e4e3d5936cbd7e5d07fbdc6e346208d3677b3910de8a7e1bed2d65a5984e2b37R303) line:
```
def <find_relative_path>(item, relative_path):
    return item[5] if len(item) >= 6 else relative_path
```
and also in this case we do not need to use if/else in [351](https://github.com/epam/cloud-pipeline/pull/3894/files#diff-e4e3d5936cbd7e5d07fbdc6e346208d3677b3910de8a7e1bed2d65a5984e2b37R351) line